src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,62 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React, { useCallback, useMemo } from 'react';
+
+export type TabType = {
+  id: string,
+  title: string,
+  content: string,
+};
+
+type Props = {
+  tabs: TabType[],
+  selectedTabId: string,
+  onTabSelected: (tab: TabType) => void
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = useMemo(
+    () => tabs.find(tab => tab.id === selectedTabId),
+    [selectedTabId],
+  ) || tabs[0];
```

Comment:
Move default value inside useMemo.
```suggestion
  const selectedTab = useMemo(
    () => tabs.find(tab => tab.id === selectedTabId),
    [selectedTabId],
  ) || tabs[0];
```